Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -1620,11 +1620,13 @@ public void RaiseManipulationCanceled(IMixedRealityInputSource source, Handednes
         /// <inheritdoc />
         public void RaiseSpeechCommandRecognized(IMixedRealityInputSource source, MixedRealityInputAction inputAction, UnityEngine.Windows.Speech.ConfidenceLevel confidence, TimeSpan phraseDuration, DateTime phraseStartTime, UnityEngine.Windows.Speech.SemanticMeaning[] semanticMeanings, string text)
```

Comment:
I think we can use our own confidence level we made for the mrtk. Let's update it to use that

Reply:
Is there a replacement for the SemanticMeanings too?